### PR TITLE
chore: UI username RTL

### PR DESCRIPTION
**Pull request by a0m0rajab**
## What does this PR do?

This fixes one of the UI issues in the RTL view that's related to the username. 
Related to: https://github.com/calcom/cal.com/issues/8151

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?
Check if the image is correct: 

image

### CODE REVIEW

File: packages/ui/components/form/inputs/Input.tsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ type AddonProps = {
 const Addon = ({ isFilled, children, className, error }: AddonProps) => (
   <div
     className={classNames(
-      "addon-wrapper border-default min-h-9 border px-3",
+      "addon-wrapper border-default min-h-9 ltr border px-3",
```
Comment (PeerRich) [inline packages/ui/components/form/inputs/Input.tsx:71]:
```suggestion
      "addon-wrapper border-default min-h-9 border px-3",
```

**Comment by PeerRich** [comment]
thanmk you! wil test and merge

**Comment by PeerRich** [comment]
can you run ´yarn format´ on root?

**Review by PeerRich — APPROVED** [review]
tested, looks good on ltr and rtl

**Comment by a0m0rajab** [comment]
@PeerRich Thank you for that! I will keep working on the issue and open the related PRs.

File: packages/ui/components/form/inputs/Input.tsx [hunk 2]
```diff
@@ -419,4 +416,4 @@ export const NumberInput = forwardRef<HTMLInputElement, InputFieldProps>(functio
       {...props}
     />
   );
-});
+});
```
Comment (PeerRich) [inline packages/ui/components/form/inputs/Input.tsx:419]:
```suggestion
});

```

**Pull request merged**